helm-charts/riff/README.md

```diff
@@ -34,6 +34,7 @@ The following lists the configurable parameters and their default values.
 | `httpGateway.image.tag`|The image tag for the http-gateway|latest|
 | `httpGateway.image.pullPolicy`|The imagePullPolicy for the http-gateway|IfNotPresent|
 | `httpGateway.service.type`|The service type used for the http-gateway|LoadBalancer|
+| `httpGateway.httpHeadersWhitelist`|A comma separated list of headers to whitelist|(none)|
```

Comment:
Nitpick -- list of _request_ headers.

Reply:
Fixed - even specified "HTTP request headers"